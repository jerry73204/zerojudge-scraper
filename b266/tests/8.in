10 10 100
3 5 5 2 1 2 3 2 1 0
3 3 1 7 4 5 6 5 4 0
5 1 7 6 9 1 2 1 6 3
8 3 5 0 9 2 9 6 7 6
4 0 4 1 1 2 2 1 8 9
9 8 0 0 1 2 0 1 7 5
0 4 5 5 9 1 9 9 7 2
3 0 6 4 8 2 1 0 3 7
8 7 9 7 0 1 9 2 4 7
8 7 9 1 6 4 3 3 4 7
0 1 0 0 1 1 0 0 1 0 1 0 0 0 1 1 0 0 0 1 1 1 1 1 0 1 0 1 1 0 1 1 0 1 1 1 0 1 0 1 0 1 1 1 1 0 0 0 0 1 1 0 0 0 0 0 0 0 1 1 0 1 1 0 1 1 1 1 0 1 0 1 0 1 1 0 0 1 0 1 1 0 1 1 1 1 1 1 0 1 0 0 0 1 1 0 0 1 1 0
