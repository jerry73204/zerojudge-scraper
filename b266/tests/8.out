10 10
7 4 3 3 4 6 1 9 7 8
7 4 2 9 1 0 7 9 7 8
7 3 0 1 2 8 4 6 0 3
2 7 9 9 1 9 5 5 4 0
5 7 1 0 2 1 0 0 8 9
9 8 1 2 2 1 1 4 0 4
6 7 6 9 2 9 0 5 3 8
3 6 1 2 1 9 6 7 1 5
0 4 5 6 5 4 7 1 3 3
0 1 2 3 2 1 2 5 5 3
